MANIFEST.in

```diff
@@ -4,19 +4,32 @@ include SYMLINK_CACHE.json
 include requirements.txt
 include shippable.yml
 include bin/ansible-test
+recursive-include docs *
+exclude docs/docsite/rst_warnings
+recursive-exclude docs/docsite/_build *
+recursive-exclude docs/docsite/extensions *.pyc *.pyo
 include examples/hosts
 include examples/ansible.cfg
 include examples/scripts/ConfigureRemotingForAnsible.ps1
 include examples/scripts/upgrade_to_ps3.ps1
-recursive-include lib/ansible/executor/powershell *
-recursive-include lib/ansible/module_utils/csharp *
-recursive-include lib/ansible/module_utils/powershell *
-recursive-include lib/ansible/modules *
-recursive-include lib/ansible/galaxy/data *
-recursive-include docs *
+recursive-include lib/ansible/executor/powershell *.ps1
+recursive-include lib/ansible/module_utils/csharp *.cs
+recursive-include lib/ansible/module_utils/powershell *.psm1
+recursive-include lib/ansible/modules/windows *.ps1
+recursive-include lib/ansible/galaxy/data *.yml *.j2 README.md ansible.cfg inventory .git_keep
+recursive-include lib/ansible/config/ *.yml
 recursive-include licenses *
 recursive-include packaging *
-recursive-include test *
+recursive-include test/cache .keep
+recursive-include test/integration *
+recursive-include test/legacy *
```

Comment:
We should probably omit these. They're not run in CI and are generally in an unknown state.

Reply:
Legacy removed from MANIFEST.in and ignored in the package-data test now.